### PR TITLE
Update dbscheme to add table for variadic signature types

### PR DESCRIPTION
**Pull request by owen-mc**
Currently there is no way to tell if a function is variadic. This PR fixes that.
- [x] Add tests.

### CODE REVIEW

**Comment by owen-mc** [comment]
@smowton I have fixed the test failures. Can you re-review?

File: ql/test/library-tests/semmle/go/Function/variadicFunctions.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	variadicDeclaredFunction() // $ isVariadic
+}
+
+func variadicDeclaredFunction(x ...int) int {
+	a := make([]int, 0, 10) // $ isVariadic
+	y := append(x, a...)    // $ isVariadic
+	print(x[0], x[1])       // $ isVariadic
+	println(x[0], x[1])     // $ isVariadic
+	variadicFunctionLiteral := func(z ...int) int { return z[1] }
+	return variadicFunctionLiteral(y...)
```
Comment (smowton) [inline ql/test/library-tests/semmle/go/Function/variadicFunctions.go:13]: We don't notice this one calls a variadic function?

Comment (owen-mc) [reply ql/test/library-tests/semmle/go/Function/variadicFunctions.go:13]: function literals are not included in `Function`.

**Comment by owen-mc** [comment]
@smowton Are you happy with this?

**Pull request merged**